modules/reitit-swagger/src/reitit/swagger.cljc

```diff
@@ -92,6 +92,7 @@
                                      (apply meta-merge (keep (comp :swagger :data) interceptors))
                                      (if coercion
                                        (coercion/get-apidocs coercion :swagger data))
+                                     (when-not (:responses data) {:responses {:default {:description ""}}})
```

Comment:
I think this should be outside of the meta-merge: if the middleware or interceptors have defined the responses, this will override them. Or, add a meta-merge hint that this values loses on merge, if such a hint exists.

Reply:
6e332e46f1f7496bf94ceb42a415bfc4ef1b49ed is an attempt to build on your remark. I'm having hard time to see where to put this default spec if not at the root of `meta-merge`, but other options I'm not aware of might be preferable.